src/services/giant_api.rs

```diff
@@ -111,3 +114,20 @@ pub fn get_or_insert_ingestion(
         }
     }
 }
+
+pub fn get_blobs_in_collection(uri: &str, collection: &str) -> Result<Vec<Blob>, CliError> {
+    let client = get_client(uri)?;
+    let encoded_collection = encode(collection);
+    let url = format!("{uri}/api/blobs?collection={encoded_collection}");
```

Comment:
Mostly just FYI, this can be done in a slightly more long winded way like so:

https://github.com/guardian/coverdrop/blob/main/common/src/api/models/api_client.rs#L84-L91

We unwrap because we've already confirmed the `base_url` is a valid base url, which is a bit ugly but oh well.